src/scene/vcVerticalMeasureTool.cpp

```diff
@@ -248,21 +271,25 @@ bool vcVerticalMeasureTool::HasLine()
 
 void vcVerticalMeasureTool::ClearPoints()
 {
-  for (size_t i = 0; i < POINTSIZE; i++)
+  for (size_t i = 0; i < udLengthOf(m_points); i++)
```

Comment:
You can also do the range-for loop here, `for (auto &point : m_points)`

Reply:
This is old code, I prefer remain it.